packages/sfc-playground/src/Header.vue

```diff
@@ -299,6 +299,15 @@ h1 img {
   color: var(--highlight);
 }
 
+.links .github {
+  color: var(--btn);
+  padding: 1px 6px;
+}
+
+.links .github:hover {
+  color: var(--highlight);
+}
```

Comment:
I think it's better to merge this into L292:300:

```css
.links > button,
.links > a {
  padding: 1px 6px;
  color: var(--btn);
}

.links > button:hover,
.links > a:hover {
  color: var(--highlight);
}
```

Reply:
done